.github/workflows/publish-pypi.yaml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    tags:
+    - '*'
```

Comment:
Could we make this an `x.y.z` tag format instead? We sometimes use tags for other stuff (e.g. marking a specific snap revision) and I don't want that to trigger the workflow even if it won't publish.

Reply:
Good point. I modified the selector following GHA's [filter pattern cheat sheet](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet).

Reply:
Thanks!